_content/news/_posts/2018-11-12-clinic.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Coaches Clinic"
+date: 2018-11-12 20:00:00
+description: Op zaterdag 22 december organiseren we 2 coach clinics voor jeugdcoaches. 
+cover: /news/img/clinic-22-12-2018.jpg
+permalink: /news/2018-11-12-clinic/
+---
+
+Op zaterdag 22 december organiseren we in samenwerking met coach Gerrit Driessens, de headcoach van Kangoeroes Willebroek en coach van het jaar 2017, 2 coaches clinics voor jeugdcoaches.
```

Comment:
### Wat is een coaches clinic?
In een clinic komt een ervaren coach uitleggen hoe hij of zij een bepaald aspect van het basketbalspel aanleert aan een team. Dit is vaak een combinatie van theoretische uitleg en voorbeeldoefeningen, gedemonstreerd door een selectie spelers.
Een mooi voorbeeld is de clinic van onze coach Philip Wolk (Dames A) op de FIBA U14 Get Together. Hij legde er uit hoe hij de dribble hand-off in een motion offense aanleert.
 <iframe width="666" height="375" src="https://www.youtube.com/embed/rwbNeHWRzrE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
De term ‘clinic’ is dus een beetje misleidend. Het is geen ‘genezing’ van een probleem maar is wel vergelijkbaar met een traditionele bedrijfspresentatie.
### Twee coaches clinics bij Basket Lummen
Op zaterdag 22 december organiseren we in samenwerking met coach Gerrit Driessens, de headcoach van Kangoeroes Willebroek en coach van het jaar 2017, 2 coaches clinics voor jeugdcoaches:
Eerste clinic: van 9u30 tot 10u45 komt coach Driessens uitleggen hoe we de trainingstijd zo goed mogelijk kunnen maximaliseren bij grote trainingsgroepen (gericht op U8 en U10) en hoe we de fysieke geletterdheid verder kunnen ontwikkelen aan de hand van spelvormen.
Tweede clinic: van 11u00 tot 12u15 komt hij uitleggen hoe we bij de U14, U16 en U18/U19 de progressie van de dribble hand-off naar pick and roll kunnen aanleren en verder ontwikkelen.

Reply:
moet de prijs voor de niet-BL-coaches nog vermeld worden?

Reply:
@DPEWillems denk het niet, we richten ons via de website vooral op onze eigen coaches, de anderen vinden die prijs op het inschrijvings formulier